snmp/datadog_checks/snmp/config.py

```diff
@@ -174,6 +174,8 @@ def __init__(
 
         self._uptime_metric_added = False
 
+        self.evaluate_bandwidth_use = instance.get('evaluate_bandwidth_use', False)
```

Comment:
I think we need to better understand if we want this feature to be available or not by default (and whether the config is needed or not).

cc @naaltman